configs/eslint-ts-overrides.json

```diff
@@ -23,7 +23,11 @@
             { "functions": false, "classes": true, "variables": true }
         ],
 
-        "@typescript-eslint/no-empty-function": "off"
+        "@typescript-eslint/no-empty-function": "off",
+
+        // See https://github.com/typescript-eslint/typescript-eslint/issues/2483
+        "no-shadow": "off",
```

Comment:
Will that disable `no-shadow` rule on non-TS files?

Reply:
Nope, the one from `@typescript-eslint` handles both file types :slightly_smiling_face: 

![image](https://user-images.githubusercontent.com/889383/111452942-51b6e080-8713-11eb-8380-b1acd6ea582f.png)

Reply:
Another question: Will it work in composer and topology?

Reply:
Not sure why it wouldn't. The ui-common repo already specifies the necessary packages and the required ESLint parser for parsing TS files, so any breaking changes have already been made. There are other `@typescript-eslint` rules already used in the file I modified :slightly_smiling_face: 

If those repos are already using this version of the config, they must have installed the necessary packages previously, and if they are not using the TS version of ui-common, then they will have to install the necessary packages when `ui-common` is upgraded in those other repos :slightly_smiling_face:

Reply:
OK, then do we have issue(s) to make cloudify-blueprint-topology and cloudify-blueprint-composer working with the latest cloudify-ui-common? I think it is quite important to be prepared for ui-common update for those repos. I will suggest Alex to put those issues to the coming sprint.

(I did quick check in cloudify-blueprint-topology and it is not just about adding missing dependencies, I see some new linter issues in topology, did not check build and tests)

Reply:
Hmm ok, sure. Here are the tickets I have created some time in the past about adding TS support in those repos (which we may as well do when upgrading `ui-common`):
1. https://cloudifysource.atlassian.net/browse/RD-1297
2. https://cloudifysource.atlassian.net/browse/RD-1298